libavfilter/vaapi_vpp.c

```diff
@@ -257,6 +257,11 @@ static const VAAPIColourProperties vaapi_colour_standard_map[] = {
     { VAProcColorStandardSMPTE170M,   6,  6,  6 },
     { VAProcColorStandardSMPTE240M,   7,  7,  7 },
     { VAProcColorStandardGenericFilm, 8,  1,  1 },
+
+//#if VA_CHECK_VERSION(2, 3, 0)
+//    { VAProcColorStandardExplicit,    9,  16, AVCOL_SPC_BT2020_NCL},
+//#endif
+
```

Comment:
why these lines was commented?

Reply:
"If explicit values are provided then you don't need it in this map." 
----from Mark T.

Reply:
> why these lines was commented?

"If explicit values are provided then you don't need it in this map."
----from Mark T.